nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-process-group.js

```diff
@@ -847,6 +870,7 @@
                     .text(function (d) {
                         return d.activeRemotePortCount;
                     });
+                transmittingCount.append("title").text("Transmitting Remote Process Groups");
```

Comment:
The `title` only needs to be appended when the corresponding `text` element is appended. This was done above. You should be able to remove the changes starting here. Let me know if you're seeing otherwise. Thanks!

Reply:
no, unfortunately every time we update the text with the function above, the title previously added is removed (I tried it). So, if we don't put it here, but where you suggest, it won't show.

Otherwise, we can change a bit the structure and include both the icon and the text in a div and put the title in the div.

Reply:
Ah sorry, you're right. I overlooked that these tips were on the counts and not the icons.